internal/cmd/dataplane.go

```diff
@@ -0,0 +1 @@
+package cmd
```

Comment:
maybe add a comment here on what's this file for (and also the boilerplate on all files)

Reply:
just a boiler plate, for when we need start to program the dataplane.